scripts/validate

```diff
@@ -1,21 +1,10 @@
 #!/bin/bash
 set -e
 
+source $(dirname $0)/version
 cd $(dirname $0)/..
 
-echo Running validation
-
-PACKAGES="$(go list ./...)"
-
-if ! command -v golangci-lint; then
-    echo Skipping validation: no golangci-lint available
-    exit
-fi
-
-echo Running validation
-
-echo Running: golangci-lint
-golangci-lint run
```

Comment:
why did we remove this?

Reply:
I removed it in favor of the golangci-lint github action. I can add this back in and simply not call the file, and run `go fmt ./...` directly from the action

Reply:
reverted